src/app/issues-viewer/hidden-groups/hidden-groups.component.ts

```diff
@@ -7,15 +8,23 @@ import { GroupBy, GroupingContextService } from '../../core/services/grouping/gr
   templateUrl: './hidden-groups.component.html',
   styleUrls: ['./hidden-groups.component.css']
 })
-export class HiddenGroupsComponent {
+export class HiddenGroupsComponent implements AfterViewInit {
   @Input() groups: Group[] = [];
 
   @ViewChild('defaultCard') defaultCardTemplate: TemplateRef<any>;
   @ViewChild('assigneeCard') assigneeCardTemplate: TemplateRef<any>;
   @ViewChild('milestoneCard') milestoneCardTemplate: TemplateRef<any>;
 
+  private currentCardTemplate$ = new BehaviorSubject<TemplateRef<any>>(null);
```

Comment:
This is fine for now. Looking at the code for grouping-context, it seems there are a lot of redundant variables which could call for potential future improvements.